detector/core/redis_store.py

```
@@ -0,0 +1,13 @@
+import json
+
+async def get_recent_candles(rdb, ticker, timeframe, limit=100):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)